src/executables/util_service.c

```diff
@@ -1442,6 +1396,61 @@ process_service (int command_type, bool process_window_service)
   return status;
 }
 
+/*
+ * get_server_names -
+ *
+ * return:
+ *
+ *      out_buffer (out):
+ */
+static void
+get_server_names (char *out_buffer)
+{
+  FILE *input;
+  char buf[4096];
+  char cmd[PATH_MAX];
+
+  if (out_buffer == NULL)
+    {
+      return;
+    }
+
+  make_exec_abspath (cmd, PATH_MAX, (char *) UTIL_COMMDB_NAME " " COMMDB_ALL_STATUS);
+  input = popen (cmd, "r");
+  if (input == NULL)
+    {
+      return;
```

Comment:
I think that a clear code should be added in case the out_buffer is not initialized.

Reply:
I've changed to allocate buffer (`name_buffer`) for server names in the get_server_names(). a function that calls the get_server_names() must free the buffer if it is not NULL.

Reply:
OK